docs/sources/tasks/nonroot.md

```diff
@@ -5,41 +5,43 @@ title: Run Alloy as a non-root user in Kubernetes
 weight: 600
 ---
 
-# Run Alloy as a non-root user in Kubernetes
+# Run {{% param "PRODUCT_NAME" %}} as a non-root user in Kubernetes
 
-The [Alloy Docker image] contains two users:
+The [{{< param "PRODUCT_NAME" >}} Docker image][image] contains two users:
 
 * A `root` user.
 * A non-root user named `alloy` with uid `473` and gid `473`.
 
-By default, the `alloy` binary runs as `root`. This is because some Alloy components like [beyla.ebpf] require root permissions.
+By default, the `alloy` binary runs as `root`. This is because some {{< param "PRODUCT_NAME" >}} components like [beyla.ebpf][] require root permissions.
 
-This page demonstrates how to configure a non-root user when deploying Alloy in Kubernetes.
+You can configure a non-root user when you deploy {{< param "PRODUCT_NAME" >}} in Kubernetes.
 
-## Configure Alloy to run as a non-root user in Kubernetes
+## Configure {{% param "PRODUCT_NAME" %}} to run as a non-root user in Kubernetes
 
 {{< admonition type="note" >}}
-Running Alloy as a non-root user will not work if you are using components like [beyla.ebpf] that require root rights.
+Running {{< param "PRODUCT_NAME" >}} as a non-root user won't work if you are using components like [beyla.ebpf][] that require root rights.
+
+[beyla.ebpf]: ../../reference/components/beyla.ebpf
 {{< /admonition >}}
 
-In order to run Alloy as a non-root user, you need to configure a [security context] for the Alloy container. If you are using the [Grafana Helm chart] you can do this by adding the following snippet to `values.yaml`:
+To run {{< param "PRODUCT_NAME" >}} as a non-root user, configure a [security context][] for the {{< param "PRODUCT_NAME" >}} container. If you are using the [Grafana Helm chart][] you can add the following snippet to `values.yaml`:
 
-```
+```yaml
 alloy:
   securityContext:
     runAsUser: 473
     runAsGroup: 473
 ```
 
-This will make the Alloy binary to run with uid 473 and gid 473 rather than as root.
+This configuration makes the {{< param "PRODUCT_NAME" >}} binary run with uid 473 and gid 473 rather than as root.
```

Comment:
should uid/gid be capitalized as UID/GID? they're technically acronyms, I believe?

Reply:
I was wondering the same... if it's general UID and GID then it's capitalised... if it's the variable it'd be `uid and `gid`.  I'll fix it up with caps